beetroot_homeworks/Homework_13/V_controller.py

```diff
@@ -0,0 +1,78 @@
+CHANNELS = ["BBC", "Discovery", "TV1000"]
+INDEX_TO_NAME = dict(enumerate(CHANNELS, start=1))
+
+
+class TVcontroller:
+    carrent_index = 1
+
+    def __init__(self, INDEX_TO_NAME) -> None:
+        self.INDEX_TO_NAME = INDEX_TO_NAME
+
+    def first_channel(self):
+        """turns on the first channel from the list."""
+
+        return self.INDEX_TO_NAME[1]
+
+    def last_channel(self):
+        """turns on the last channel from the list."""
+
+        return self.INDEX_TO_NAME[len(self.INDEX_TO_NAME)]
```

Comment:
You can use negative index to return the last element instead of computing `len`.